src/current/_includes/v23.2/cdc/mux-rangefeed.md

```diff
@@ -0,0 +1,28 @@
+`MuxRangefeed` is a subsystem that improves the performance of rangefeeds with scale. It significantly reduces the overhead of running {% if page.name == "create-and-configure-changefeeds.md" %} [rangefeeds](#enable-rangefeeds) {% else %} [rangefeeds]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds). {% endif %} Without `MuxRangefeed` enabled the number of RPC streams is proportional with the number of ranges in a table. For example, a large table could have tens of thousands of ranges. With `MuxRangefeed` enabled, this proportion improves so that the number of RPC streams is relative to the number of nodes in a cluster.
+
+You can enable its functionality with the `changefeed.mux_rangefeed.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}).
```

Comment:
call out that this will be the default in 24.1; and that we encourage customers to try this in 23.2

Reply:
Done!